fire/cli/niv/_udtræk_observationer.py

```diff
@@ -137,17 +137,24 @@ def valider_om_projekt_er_okay(ctx, param, value):
 @click.option(
     "-df",
     "--fra",
-    help="Hent observationer fra og med denne dato.",
+    help=f"Hent observationer fra og med denne dato. Angives på formen {DATE_FORMAT}.",
     required=False,
     type=Datetime(format=DATE_FORMAT),
 )
 @click.option(
     "-dt",
     "--til",
-    help="Hent observationer til, men ikke med, denne dato.",
+    help="Hent observationer til, men ikke med, denne dato. Angives på formen {DATE_FORMAT}.",
     required=False,
     type=Datetime(format=DATE_FORMAT),
 )
+@click.option(
+    "-ao",
```

Comment:
Bare en kommentar: I UNIX et al. er dette énsbetydende med, at man har slået to flag `-a` og `-o` sammen, så der er måske en risiko for, at dette vil forvirre. Men ud over dét, så give to bogstaver mere tydelighed, når man kender den lange version af flaget.

Reply:
Samme program har også en `-df` og `-dt` så tænker det er okay her. Men det er rigtigt at det er en potentiel kilde til forvirring for den garvede unix-bruger. Dem har vi heldigvis ikke så mange af :-)